### PR TITLE
Use pulp_user instead of 'apache'

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: ansible/roles/dev/tasks/main.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+- name: create pulp user
+  user:
+    name: pulp
```
Comment (daviddavis) [inline ansible/roles/dev/tasks/main.yml]: This is temporary until https://pulp.plan.io/issues/3065 is solved.

**Comment by daviddavis** [comment]
I don't think this is a stopgap as the work here needs to be done at some point before issue 3065 is solved. I do agree that having to create a pulp user temporarily is suboptimal though so I am happy to close this out and add another checklist item to issue 3065.

File: ansible/roles/dev/tasks/main.yml [hunk 2]
```diff
@@ -91,16 +95,16 @@
   file:
     path: /var/lib/pulp
     state: directory
-    owner: apache
-    group: apache
+    owner: pulp
```
Comment (asmacdo) [inline ansible/roles/dev/tasks/main.yml]: I think this will do the trick:
s/pulp/{{ pulp_user }}/

If that works, I'm on board.

**Comment by daviddavis** [comment]
@asmacdo Updated the PR based on your suggestion. I tested it out and it works well. Thanks.

**Pull request merged**